Login NIH Fix:

ogwuazors opened the pull request:
- other idps disabled for now. can be removed later. will likely need a story to detail the removal of options.

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

ogwuazors committed:
Login NIH Fix:
- other idps disabled for now. can be removed later. will likely need a story to detail the removal of options.

iksheth commented on src/bento/siteWideConfig.js:
```diff
@@ -5,4 +5,17 @@ export default {
   enableAuthentication: true,
   // List for options for authentication empty array defaults to google
   authEndPoint: ['google'], // authEndPoint: []
```
@ogwuazors Can you remove "authEndPoint" it's changed to "authProviders" now.

iksheth replied:
@ogwuazors Can we remove "authEndPoint" property from here. It's no longer used.

iksheth commented on src/bento/userLoginData.js:
```diff
@@ -0,0 +1,26 @@
+  google: {
+    key: 'google',
+    icon: 'https://raw.githubusercontent.com/CBIIT/datacommons-assets/main/bento/images/icons/png/google.png',
+    loginButtonText: 'Sign in with Google',
```
Can we remove "Sign in with" or "Sign in" from each button?

ogwuazors replied:
Buttons should now only say the name of the IDP being used for logging in

iksheth commented on src/components/Auth/AuthProvider.js:
```diff
@@ -0,0 +1,146 @@
+import GET_USER_DETAILS from '../../bento/authProviderData';
+
+const AUTH_API = env.REACT_APP_AUTH_SERVICE_API;
+const GOOGLE_CLIENT_ID = env.REACT_APP_GOOGLE_CLIENT_ID;
```
Can you add support for situation when "REACT_APP_GOOGLE_CLIENT_ID"  is undefined? 

For More info: https://github.com/CBIIT/bento-frontend/pull/672/commits/d94c384d4bde7d91078b15f5a8f3d12614856b89

ogwuazors replied:
Made the change for this.

ogwuazors committed:
Login NIH Fix:
- other idps disabled for now. can be removed later. will likely need a story to detail the removal of options.
- removed authEndpoint

ogwuazors committed:
Login NIH Fix:
- other idps disabled for now. can be removed later. will likely need a story to detail the removal of options.
- removed authEndpoint
- added default client ids

iksheth reviewed (approved):
Looks good to me 👍🏻
